hera_cal/nucal.py

```diff
@@ -621,4 +684,281 @@ def build_nucal_wgts(data_flags, data_nsamples, autocorrs, auto_flags, radial_re
         data_is_redsol=data_is_redsol, gain_flags=gain_flags, tol=tol, antpos=antpos
     )
 
-    return wgts
\ No newline at end of file
+    return wgts
+
+def _linear_fit(XTX, Xy, solver='lu_solve', tol=1e-15):
```

Comment:
This feels like it probably belongs in hera_filters, rather than here, but I'm fine including it here for now if you want to make an issue to move it.

Reply:
Will do. I'll make an issue after this is merged in